src/components/Swiper/styled.tsx

```diff
@@ -0,0 +1,116 @@
+import styled, { createGlobalStyle, css } from 'styled-components';
+import { em, transparentize } from 'polished';
+import { Swiper } from 'swiper/react';
+
+import swiper from '../../../node_modules/swiper/swiper.min.css';
+import swiperNavigation from '../../../node_modules/swiper/components/navigation/navigation.min.css';
+import swiperPagination from '../../../node_modules/swiper/components/pagination/pagination.min.css';
+import { GoldDark } from '../../modules/themes/themes/GoldDark';
+import { SIZES } from '../internal/useWindowSize';
+
+import prev from './prev.svg';
+import next from './next.svg';
+
+const SLIDE_WIDTH = 270;
+const SLIDE_HEIGHT = 152;
+export const MARGIN_WIDTH = 40;
+
+export const Styles = createGlobalStyle`
+  ${swiper};
+  ${swiperNavigation};
+  ${swiperPagination};
+`;
+
+const buttons = css`
+  width: ${({ theme }) => em(theme.honeycomb.size.large)};
+  height: ${em(SLIDE_HEIGHT)};
+  top: 0;
+  margin: 0;
+  cursor: pointer;
+  background-color: ${transparentize(0.7, GoldDark.honeycomb.color.bg.tooltip.normal)};
+  background-repeat: no-repeat;
+  background-position: center;
+`;
+
+const navigation = css`
+  .swiper-button-prev {
```

Comment:
It is pretty optional but I tend to have something like this:
It's fine if you keep it.

```
.swiper-button-prev,
.swiper-button-next {
    ${buttons};
    ::after {
      display: none;
    }
 }
```

Reply:
Done!